Resources/English.lproj/MainMenu.xib

```diff
@@ -298,8 +299,8 @@
                 </menuItem>
             </items>
         </menu>
-        <window title="Hermes" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" frameAutosaveName="Preferences" animationBehavior="default" id="371" userLabel="Main Window" customClass="HermesMainWindow">
-            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" texturedBackground="YES" unifiedTitleAndToolbar="YES"/>
+        <window title="Hermes" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" frameAutosaveName="Preferences" animationBehavior="default" titlebarAppearsTransparent="YES" id="371" userLabel="Main Window" customClass="HermesMainWindow">
```

Comment:
`unifiedTitleAndToolbar` is deprecated. Setting `titlebarAppearsTransparent` to `YES` achieves the same UI as before.

Reply:
Where do you see this? Even in the 10.14 SDK NSWindow.h I see no deprecation warnings:

```
    /* Specifies a window whose titlebar and toolbar have a unified look - that is, a continuous background. Under the titlebar and toolbar a horizontal separator line will appear.
     */
    NSWindowStyleMaskUnifiedTitleAndToolbar = 1 << 12,
```

Reply:
OK, I think I see what you meant — NSWindowStyleMaskTexturedBackground is “to be deprecated” and this is what breaks when built with the 10.14 SDK. We may want to draw our own gradient, use some of the fancy new style stuff so dark mode looks awesome, or figure out another way to handle this.